lib/core/test/integration/Test/Integration/Framework/DSL.hs

```diff
@@ -829,6 +872,12 @@ deleteWalletEp w =
     , "v2/wallets/" <> w ^. walletId
     )
 
+getWalletUtxoEp :: ApiWallet -> (Method, Text)
+getWalletUtxoEp w =
+    ( "GET"
+    , "v2/wallets/" <> w ^. walletId <> "/statistics/utxos"
```

Comment:
I wonder if this is not inconsistence that we have endpoint with `utxos` and in CLI we use `utxo`.

Reply:
We actually use plural forms for resources in the API endpoints, which is kinda more RESTful. The CLI audience is slightly different though and uses a different style (singular) which usually reads better for command line tools :+1: